docs/tutorial.md

```diff
@@ -0,0 +1,138 @@
+4. Take the `accessKeyId` and `secretAccessKey` from step 2 and add a local `~/.anycloud/credentials.json` file with the following contents:
+
+```javascript
+{
+  "aws-personal": {
+    "cloudProvider": "AWS",
+    "credentials": {
+      "accessKeyId": "#####################",
+      "secretAccessKey": "###################"
+    },
+  }
+}
+```
+
+## Configure your project
+
+1. Initialize a `git` repository
+
+```bash
+git init
+git add -A
+git commit -m "Initial commit"
+```
+
+2. Initialize your `package.json` and install `express`
+
+```bash
+npm init
+npm install express --save
+```
+
+3. Define an HTTP server listening on port `8088` in an `index.js` file:
+
+```javascript
+const express = require('express')
+const app = express()
+const port = process.env.PORT
+
+app.get('/', (req, res) => {
+  res.send('Hello World!')
+})
+
+app.listen(port, () => {
+  console.log(`Example app listening at http://localhost:${port}`)
+})
+```
+
+4. Define the `Dockerfile` like this: 
+
+```bash
+FROM node:lts
+
+ENV PORT 8088
+
+COPY . .
+
+RUN npm install
+CMD node index.js
+```
+
+5. Test the `Dockerfile` locally by installing [Docker Desktop](https://www.docker.com/products/docker-desktop), building the Docker image and then running the server within the container
+
+```bash
+$ docker build -t anycloud/app .
+$ docker run -p 8088:8088 -d anycloud/app:latest
+$ curl localhost:8088
+```
+
+Which should return `Hello World!`
+
+ 6. Define your deployment configuration in `anycloud.json` like this:
+
+```javascript
+{
+  "staging": [{
+    "credentials": "aws-personal",
+    "region": "us-west-1",
+    "vmType": "t3.medium"
+  }]
+}
+```
+
+We are referencing the previously defined credentials so make sure that the `credentials` value matches the key in `~/.anycloud/credentials.json`
+
+## Deploy an app
+
+1. Make sure you [install the AnyCloud CLI](about.md#cli-installation). Now deploy your Node.js server to your AWS account using the AnyCloud CLI:
+
+```bash
+$ anycloud new staging
+▇ Creating new app
+```
+
+It might take a few minutes for your app to start while the virtual machine is provisioned and upgraded. The AnyCloud CLI will generate a random app name for you, but [a custom app name can be used](how-to-guides/custom-name.md). 
+
+2. Check the status of your application:
+
+```bash
+$ anycloud info
+Status of all apps deployed:
+
+┌─────────────────┬─────────────────────────────────────────┬───────────────┬──────┬─────────┐
+│ App Id          │ Url                                     │ Deploy Config │ Size │ Version │
+├─────────────────┼─────────────────────────────────────────┼───────────────┼──────┼─────────┤
+│ maroon-egret-25 │ https://maroon-egret-25.anycloudapp.com │ staging       │ 1    │ v0.1.34 │
```

Comment:
Two things:

1. The actual version they'll see is v0.1.35 as of right now
2. This column actually doesn't make any sense to the anycloud users. Should we even have it?

Reply:
This table is the same for `alan deploy` as for `anycloud`. We could remove it in the former case, but then we add more special casing. Also this *might* make debugging for us easier when a user is having a problem and it could also prompt the users to upgrade.

Reply:
btw @dfellis in order to display `status` in the column we need to call the status/health control port outside of the upgrading a cluster, can you address https://github.com/alantech/deploy/pull/62#discussion_r599958706 in a follow up PR?

Reply:
https://github.com/alantech/deploy/issues/64